alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -118,11 +119,14 @@ object DocumentLevelMonitorRunner : MonitorRunner() {
 
         try {
             // Resolve all passed indices to concrete indices
-            val indices = IndexUtils.resolveAllIndices(
+            val concreteIndices = IndexUtils.resolveAllIndices(
                 docLevelMonitorInput.indices,
                 monitorCtx.clusterService!!,
                 monitorCtx.indexNameExpressionResolver!!
             )
+            if (concreteIndices.isEmpty()) {
+                throw IndexNotFoundException(docLevelMonitorInput.indices[0])
```

Comment:
1. can we add error log with monitor Id and indices list which were not found
2. in exception why are we logging only 0th index element of `indices` variable? can we log all elements in array/list

Reply:
changed it now. it was copied from eariler logic.